src/components/ux-button/ux-button.steps.js

```diff
@@ -32,22 +32,21 @@ module.exports = function () {
 	this.Then(/^the element "([^"]*)" should have the "([^"]*)" of "([^"]*)"$/,
 		function (semanticName, attribName, attribValue, callback) {
 
-		var self = this;
-
-		this.client.getAttribute(
-			component[semanticName].selector,
-			component[semanticName].attribs[attribName],
-			function (err, attr) {
-				try {
-					self.assert.deepEqual(attr, attribValue);
-					callback();
-				} catch (e) {
-					callback.fail(e.name + ' ' + e.message);
+			var self = this;
+
+			this.client.getAttribute(
+				component[semanticName].selector,
+				component[semanticName].attribs[attribName],
+				function (err, attr) {
+					try {
+						self.assert.deepEqual(attr, attribValue);
+						callback();
+					} catch (e) {
+						callback.fail(e.name + ' ' + e.message);
+					}
 				}
+			);
 
-			}
-		);
-
-	});
+		});
```

Comment:
If you line the curly braces (}) you should also align the round brackets ie split this line into to.